kotlin/android/app/src/main/java/dev/firezone/android/core/data/Repository.kt

```diff
@@ -13,11 +13,7 @@ interface Repository {
 
     fun getDefaultConfig(): Flow<Config>
 
-    fun saveSettings(
-        authBaseUrl: String,
-        apiUrl: String,
-        logFilter: String,
-    ): Flow<Unit>
+    fun saveConfig(value: Config): Flow<Unit>
```

Comment:
Not sure if we're moving towards "config" or "settings" but everything else in this file was calling it a "config"